Added a reminder command

Martian-0007 opened the pull request:
formated with `black .`, thats why all the changed files

Martian-0007 committed:
Added a reminder command

formated with `black .`, thats why all the changed files

TheXer commented:
Využít skvělou utilitu, co jsem našel na netu - https://github.com/scrapinghub/dateparser

Dokáže fungovat v českém jazyce, příklad použití je v commitu 3958343f41aeb487458e8840ff43b573f75691d9 kde toto je řešený. Zkus se na to podívat a upravit parametry

TheXer committed:
Merge branch 'jachym_2.0' into reminder-branch

TheXer commented:
Spraveny konflikty mezi verzemi

Martian-0007 committed:
Add dateparser and countdown function

Added dateparser functionality and returns in how long the reminder will be triggered

Martian-0007 committed:
formated with black
